### PR TITLE
tidb-in-kubernets: refine kubernetes and tidb deployment docs

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
1. add a note for namespace in general Kubernetes deployment doc
2. add kind deployment in the overview page

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1970

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

**Comment by anotherrachel** [comment]
@yikeke PTAL

File: dev/tidb-in-kubernetes/tidb-operator-overview.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ TiDB Operator provides several ways to deploy TiDB clusters in Kubernetes:
 
 + For test environment:
 
+    - [kind](/dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md): Deploy TiDB clusters in Kubernetes using kind
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/tidb-operator-overview.md]:
```suggestion
    - [kind](/dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md): Deploy TiDB clusters in local Kubernetes using kind
```

File: dev/tidb-in-kubernetes/deploy/general-kubernetes.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ After you deploy and configure TiDB Operator, deploy the TiDB cluster using the
 helm install pingcap/tidb-cluster --name=<release-name> --namespace=<namespace> --version=<chart-version> -f /home/tidb/<release-name>/values-<release-name>.yaml
 ```
 
+> **Note:**
+>
+> [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) is a virtual cluster backed by the same physical cluster. You can give it a name that is easy to remember, such as the same name as `release-name`.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/general-kubernetes.md]:
```suggestion
> A [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) is a virtual cluster backed by the same physical cluster. You can give it a name that is easy to memorize, such as the same name as `release-name`.
```

**Comment by anotherrachel** [comment]
All comments addressed and v3.0 updated. PTAL, thanks! @yikeke

**Pull request merged**